programs/ziti-edge-tunnel/ziti-edge-tunnel.c

```diff
@@ -110,16 +110,35 @@ static int start_cmd_socket(uv_loop_t *l) {
     if (uv_is_active((const uv_handle_t *) &cmd_server)) {
         return 0;
     }
+#if _WIN32
+    static char sockfile[] = "\\\\.\\pipe\\ziti-edge-tunnel.sock";
```

Comment:
it might be worthwhile to namespace these just for clarity's sake. I'd also used a pipe for one-way events (from the service to a consumer). just a passing thought/consideration